resources/views/bread/browse.blade.php

```diff
@@ -178,7 +178,23 @@
                                                     <span class="badge badge-lg" style="background-color: {{ $data->{$row->field} }}">{{ $data->{$row->field} }}</span>
                                                 @elseif($row->type == 'text')
                                                     @include('voyager::multilingual.input-hidden-bread-browse')
-                                                    <div>{{ mb_strlen( $data->{$row->field} ) > 200 ? mb_substr($data->{$row->field}, 0, 200) . ' ...' : $data->{$row->field} }}</div>
+
+                                                    @php
+                                                        $content = mb_strlen( $data->{$row->field} ) > 200 ? mb_substr($data->{$row->field}, 0, 200) . ' ...' : $data->{$row->field};
+
+                                                        $view = null;
+                                                        if (property_exists($row->details, 'link')) {
+                                                            $view = collect($row->details->link)->first(function($view, $permission) use ($dataType) {
+                                                                return Auth::user()->can($permission, app($dataType->model_name));
+                                                            });
+                                                        }
+                                                    @endphp
+
+                                                    @if ($view)
+                                                        <a href="{{ route('voyager.' .$dataType->slug. '.' .$view, [ 'id' => $data->id ]) }}">{{ $content }}</a>
```

Comment:
To be ready for Laravel 6 route should be:
```php
route('voyager.' .$dataType->slug. '.' .$view, $data->id)
```

Reply:
done

Reply:
I think documentation needs to be updated to explain this new parameter.
Probably add a voyager/docs/bread/formfields/text.md like number.md

Another thing that needs to be changed that I didn't notice before is the permission check, you should use $data instead of $dataType because, depending on policy, permissions can change from one element to another.

As example in a standard new install if you give a user only browse permission for users BREAD, that user will be able to browse users and edit and view itself but not other users.
Using $data instead of $dataType gets the same behaviour.
![Schermata da 2019-10-05 10-18-29](https://user-images.githubusercontent.com/6875243/66252259-92a67980-e759-11e9-99a5-847d30d8b099.png)

P.S.
I would like to clarify that I'm a contributor just like you, so whatever I say is just my opinion don't feel compelled to implement it, you might want to wait @emptynick feedback before working on changes.

Reply:
```php
$link = collect($row->details->link)->first(function($view, $permission) use ($data) {
    return Auth::user()->can($permission, $data);
});
```

Reply:
@MrCrayon hmmmm, `$data` is undefined in the blade and the Controller. Did you mean a different variable?